<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plotX" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotY" native="true">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>39</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotXY" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
